[Expenditures] java.lang.NullPointerException: Add file fails when no actual file is selected

Caused by: java.lang.NullPointerException: Content byte[] is null
at pt.ist.fenixframework.plugins.fileSupport.domain.GenericFile.init(GenericFile.java:36)
at pt.ist.expenditureTrackingSystem.domain.acquisitions.AcquisitionProposalDocument.<init>(AcquisitionProposalDocument.java:66)
    ... 99 more

--- a/expenditure-tracking/src/main/webapp/WEB-INF/expenditure-tracking/schemas-config.xml
+++ b/expenditure-tracking/src/main/webapp/WEB-INF/expenditure-tracking/schemas-config.xml
@@ -589,7 +589,7 @@
 	type="pt.ist.expenditureTrackingSystem.presentationTier.util.FileUploadBean"
 	bundle="ACQUISITION_RESOURCES">
 	<slot name="inputStream" key="acquisitionProcess.label.acquisitionRequestDocument"
-		bundle="ACQUISITION_RESOURCES">
+		bundle="ACQUISITION_RESOURCES" required="true">
 		<property name="fileNameSlot" value="filename" />
 		<property name="size" value="30" />
 	</slot>
@@ -603,7 +603,7 @@
 		<property name="size" value="50" />
 	</slot>
 	<slot name="inputStream" key="acquisitionProcess.label.document"
-		bundle="ACQUISITION_RESOURCES">
+		bundle="ACQUISITION_RESOURCES" required="true">
 		<property name="fileNameSlot" value="filename" />
 		<property name="size" value="50" />
 	</slot>
@@ -1396,7 +1396,7 @@
 	<slot name="proposalID" key="acquisitionProcess.label.proposalID.optional"
 		bundle="ACQUISITION_RESOURCES"
 		required="true" />
-	<slot name="inputStream" key="label.document" bundle="WORKFLOW_RESOURCES">
+	<slot name="inputStream" key="label.document" bundle="WORKFLOW_RESOURCES" required="true">
 		<property name="fileNameSlot" value="filename" />
 		<property name="size" value="50" />
 	</slot>

--- a/expenditure-tracking/src/main/webapp/statistics/showStatisticsRefundProcess.jsp
+++ b/expenditure-tracking/src/main/webapp/statistics/showStatisticsRefundProcess.jsp
@@ -8,7 +8,6 @@
 
 <fr:edit id="yearBean"
 		name="yearBean"
-		schema="pt.ist.expenditureTrackingSystem.presentationTier.actions.statistics.YearBean"
 		action="statistics.do?method=showRefundProcessStatistics">
 		<fr:layout name="tabular">
 			<fr:property name="classes" value="form"/>

--- a/expenditure-tracking/src/main/webapp/statistics/showStatisticsReports.jsp
+++ b/expenditure-tracking/src/main/webapp/statistics/showStatisticsReports.jsp
@@ -8,7 +8,6 @@
 
 <fr:edit id="yearBean"
 		name="yearBean"
-		schema="pt.ist.expenditureTrackingSystem.presentationTier.actions.statistics.YearBean"
 		action="statistics.do?method=showStatisticsReports">
 		<fr:layout name="tabular">
 			<fr:property name="classes" value="form"/>

--- a/expenditure-tracking/src/main/webapp/statistics/showStatisticsSimplifiedProcess.jsp
+++ b/expenditure-tracking/src/main/webapp/statistics/showStatisticsSimplifiedProcess.jsp
@@ -8,7 +8,6 @@
 
 <fr:edit id="yearBean"
 		name="yearBean"
-		schema="pt.ist.expenditureTrackingSystem.presentationTier.actions.statistics.YearBean"
 		action="statistics.do?method=showSimplifiedProcessStatistics">
 		<fr:layout name="tabular">
 			<fr:property name="classes" value="form"/>

